core/cli_entry.py

```diff
@@ -77,4 +79,27 @@ def validate_and_upload_single_entry(directory):
 
     logging.info("Samples in directory '{}' have finished uploading!".format(directory))
 
+    return exit_success()
+
+
+def exit_error():
```

Comment:
Comments for all these new methods please.

Reply:
added